logging/google/cloud/logging/_gax.py

```diff
@@ -420,225 +419,14 @@ def metric_delete(self, project, metric_name):
-        }
-
-    return mapping
-
-
-def _http_request_mapping_to_pb(info, request):
-    """Helper for _log_entry_mapping_to_pb
-
-    Performs "impedance matching" between the protobuf attrs and the keys
-    expected in the JSON API.
-    """
-    optional_request_keys = {
-        'requestMethod': 'request_method',
-        'requestUrl': 'request_url',
-        'status': 'status',
-        'referer': 'referer',
-        'userAgent': 'user_agent',
-        'cacheHit': 'cache_hit',
-        'requestSize': 'request_size',
-        'responseSize': 'response_size',
-        'remoteIp': 'remote_ip',
-    }
-    for key, pb_name in optional_request_keys.items():
-        if key in info:
-            setattr(request, pb_name, info[key])
-
-
-def _log_operation_mapping_to_pb(info, operation):
-    """Helper for _log_entry_mapping_to_pb
-
-    Performs "impedance matching" between the protobuf attrs and the keys
-    expected in the JSON API.
-    """
-    operation.producer = info['producer']
-    operation.id = info['id']
-
-    if 'first' in info:
-        operation.first = info['first']
-
-    if 'last' in info:
-        operation.last = info['last']
-
-
 def _log_entry_mapping_to_pb(mapping):
     """Helper for :meth:`write_entries`, et aliae
 
-    Performs "impedance matching" between the protobuf attrs and the keys
-    expected in the JSON API.
+    Performs "impedance matching" between the protobuf attrs and
+    the keys expected in the JSON API.
     """
-    # pylint: disable=too-many-branches
     entry_pb = LogEntry()
-
-    optional_scalar_keys = {
-        'logName': 'log_name',
-        'insertId': 'insert_id',
-        'textPayload': 'text_payload',
-    }
-
-    for key, pb_name in optional_scalar_keys.items():
-        if key in mapping:
-            setattr(entry_pb, pb_name, mapping[key])
-
-    if 'resource' in mapping:
-        entry_pb.resource.type = mapping['resource']['type']
-
-    if 'severity' in mapping:
-        severity = mapping['severity']
-        if isinstance(severity, str):
-            severity = LogSeverity.Value(severity)
-        entry_pb.severity = severity
-
     if 'timestamp' in mapping:
-        timestamp = _datetime_to_pb_timestamp(mapping['timestamp'])
-        entry_pb.timestamp.CopyFrom(timestamp)
-
-    if 'labels' in mapping:
-        for key, value in mapping['labels'].items():
-            entry_pb.labels[key] = value
-
-    if 'jsonPayload' in mapping:
-        ParseDict(mapping['jsonPayload'], entry_pb.json_payload)
-
-    if 'protoPayload' in mapping:
-        ParseDict(mapping['protoPayload'], entry_pb.proto_payload)
-
-    if 'httpRequest' in mapping:
-        _http_request_mapping_to_pb(
-            mapping['httpRequest'], entry_pb.http_request)
-
-    if 'operation' in mapping:
-        _log_operation_mapping_to_pb(
-            mapping['operation'], entry_pb.operation)
-
+        mapping['timestamp'] = _datetime_to_rfc3339(mapping['timestamp'])
+    ParseDict(mapping, entry_pb)
```

Comment:
Does `ParseDict` actually know about the `multi_word_names` -> `multiWordNames` mappings?